packages/ui/react-ui/src/types.ts

```diff
@@ -0,0 +1,52 @@
+import { CSSProperties, MouseEvent } from "react";
+
+export enum clientNames {
+    reactUi = "client-sdk-react-ui",
+}
+
+export enum baseUrls {
+    prod = "https://www.crossmint.io",
+    dev = "http://localhost:3001",
+}
+
+export enum customHeaders {
+    clientVersion = "X-Client-Version",
+    clientName = "X-Client-Name",
+}
+
+enum theme {
+    light = "light",
+    dark = "dark",
+}
+
+export enum payButtonTypes {
+    solanaCandyMachine = "solana-candy-machine",
```

Comment:
In general we should follow [this](https://github.com/Paella-Labs/crossbit-main/blob/dc6f1bf0b25d6a819f542ad630027f49d1be4b70/models/ClientIntegration.ts#L7) enum